src/disc/disc.py

```diff
@@ -11,6 +11,44 @@
 ADMINISTRATOR = 1 << 3
 VIEW_CHANNEL = 1 << 10
 
+class User:
+    def __init__(self, id: str, username: str, discriminator: str, global_name: str=None, avatar: str=None) -> None:
+        self.id = id
+        self.username = username
+        self.discriminator = discriminator
+        self.global_name = global_name
+        self.avatar = avatar
+    
+    def __str__(self) -> str:
+        return f"{self.username}, {self.id}"
+    
+    def __repr__(self) -> str:
+        return f"{self.username}, {self.id}"
```

Comment:
Instead of overriding the defaults, should we make a custom function (say `def compact_str(): ...`) so that any debug printing will show all the data? I'm not sure it would make sense.

Reply:
After updating to use dataclasses, the autogenerated `__repr__` does this for us.